<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>916</width>
    <height>257</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Time Watcher</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout_4">
    <item>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="1" column="2">
       <layout class="QGridLayout" name="gridLayout">
        <item row="1" column="0">
         <widget class="QComboBox" name="selectUserName">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
         </widget>
        </item>
        <item row="1" column="3">
         <widget class="QSpinBox" name="timeLeft">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="buttonSymbols">
           <enum>QAbstractSpinBox::PlusMinus</enum>
          </property>
         </widget>
        </item>
        <item row="1" column="2">
         <widget class="QLabel" name="label_10">
          <property name="text">
           <string>Time Left for today</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <spacer name="horizontalSpacer_4">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item row="1" column="5">
       <widget class="QLabel" name="label_11">
        <property name="text">
         <string>User Account is</string>
        </property>
       </widget>
      </item>
      <item row="1" column="3">
       <widget class="QPushButton" name="pushButton">
        <property name="text">
         <string>Apply New Time</string>
        </property>
       </widget>
      </item>
      <item row="1" column="4">
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="1" column="6">
       <widget class="QPushButton" name="buttonAccountState">
        <property name="text">
         <string>UnLocked</string>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="7">
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="1" column="1">
       <widget class="QLabel" name="label_userName">
        <property name="text">
         <string>Current User</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <spacer name="verticalSpacer_2">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <layout class="QGridLayout" name="gridLayout_4">
      <property name="horizontalSpacing">
       <number>6</number>
      </property>
      <item row="0" column="3">
       <widget class="QLabel" name="labelMonday">
        <property name="text">
         <string>Monday</string>
        </property>
       </widget>
      </item>
      <item row="1" column="9">
       <widget class="QSpinBox" name="sunTime">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="value">
         <number>60</number>
        </property>
       </widget>
      </item>
      <item row="1" column="8">
       <widget class="QSpinBox" name="satTime">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="value">
         <number>60</number>
        </property>
       </widget>
      </item>
      <item row="1" column="3">
       <widget class="QSpinBox" name="monTime">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="value">
         <number>60</number>
        </property>
       </widget>
      </item>
      <item row="1" column="4">
       <widget class="QSpinBox" name="tueTime">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="value">
         <number>60</number>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QCheckBox" name="limitTimeAccess">
        <property name="text">
         <string>Limit Daily Access</string>
        </property>
       </widget>
      </item>
      <item row="0" column="7">
       <widget class="QLabel" name="labelFri">
        <property name="text">
         <string>Friday</string>
        </property>
       </widget>
      </item>
      <item row="2" column="8">
       <widget class="QTimeEdit" name="startTimeSat">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="time">
         <time>
          <hour>8</hour>
          <minute>0</minute>
          <second>0</second>
         </time>
        </property>
       </widget>
      </item>
      <item row="2" column="3">
       <widget class="QTimeEdit" name="startTimeMon">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="time">
         <time>
          <hour>8</hour>
          <minute>0</minute>
          <second>0</second>
         </time>
        </property>
       </widget>
      </item>
      <item row="1" column="5">
       <widget class="QSpinBox" name="wedTime">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="value">
         <number>60</number>
        </property>
       </widget>
      </item>
      <item row="0" column="6">
       <widget class="QLabel" name="labelThu">
        <property name="text">
         <string>Thursday</string>
        </property>
       </widget>
      </item>
      <item row="0" column="8">
       <widget class="QLabel" name="labelSat">
        <property name="text">
         <string>Saturday</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QCheckBox" name="setLoginTimes">
        <property name="text">
         <string>Set Login Times Per Day</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QCheckBox" name="limitLoginTimes">
        <property name="text">
         <string>Limit Login Times</string>
        </property>
       </widget>
      </item>
      <item row="4" column="5">
       <widget class="QTimeEdit" name="endTimeWed">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="time">
         <time>
          <hour>22</hour>
          <minute>0</minute>
          <second>0</second>
         </time>
        </property>
       </widget>
      </item>
      <item row="1" column="7">
       <widget class="QSpinBox" name="friTime">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="value">
         <number>60</number>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Start</string>
        </property>
       </widget>
      </item>
      <item row="4" column="4">
       <widget class="QTimeEdit" name="endTimeTue">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="time">
         <time>
          <hour>22</hour>
          <minute>0</minute>
          <second>0</second>
         </time>
        </property>
       </widget>
      </item>
      <item row="2" column="4">
       <widget class="QTimeEdit" name="startTimeTue">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="time">
         <time>
          <hour>8</hour>
          <minute>0</minute>
          <second>0</second>
         </time>
        </property>
       </widget>
      </item>
      <item row="4" column="2">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>End</string>
        </property>
       </widget>
      </item>
      <item row="0" column="4">
       <widget class="QLabel" name="labelTuesday">
        <property name="text">
         <string>Tuesday</string>
        </property>
       </widget>
      </item>
      <item row="4" column="3">
       <widget class="QTimeEdit" name="endTimeMon">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="time">
         <time>
          <hour>22</hour>
          <minute>0</minute>
          <second>0</second>
         </time>
        </property>
       </widget>
      </item>
      <item row="1" column="6">
       <widget class="QSpinBox" name="thuTime">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="value">
         <number>60</number>
        </property>
       </widget>
      </item>
      <item row="2" column="9">
       <widget class="QTimeEdit" name="startTimeSun">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="time">
         <time>
          <hour>8</hour>
          <minute>0</minute>
          <second>0</second>
         </time>
        </property>
       </widget>
      </item>
      <item row="2" column="6">
       <widget class="QTimeEdit" name="startTimeThu">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="time">
         <time>
          <hour>8</hour>
          <minute>0</minute>
          <second>0</second>
         </time>
        </property>
       </widget>
      </item>
      <item row="0" column="9">
       <widget class="QLabel" name="labelSun">
        <property name="text">
         <string>Sunday</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QCheckBox" name="setTimePerDay">
        <property name="text">
         <string>Set Allowed Time Per Day</string>
        </property>
       </widget>
      </item>
      <item row="4" column="9">
       <widget class="QTimeEdit" name="endTimeSun">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="time">
         <time>
          <hour>22</hour>
          <minute>0</minute>
          <second>0</second>
         </time>
        </property>
       </widget>
      </item>
      <item row="4" column="7">
       <widget class="QTimeEdit" name="endTimeFri">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="time">
         <time>
          <hour>22</hour>
          <minute>0</minute>
          <second>0</second>
         </time>
        </property>
       </widget>
      </item>
      <item row="2" column="5">
       <widget class="QTimeEdit" name="startTimeWed">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="time">
         <time>
          <hour>8</hour>
          <minute>0</minute>
          <second>0</second>
         </time>
        </property>
       </widget>
      </item>
      <item row="0" column="5">
       <widget class="QLabel" name="labelWed">
        <property name="text">
         <string>Wednesday</string>
        </property>
       </widget>
      </item>
      <item row="2" column="7">
       <widget class="QTimeEdit" name="startTimeFri">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="time">
         <time>
          <hour>8</hour>
          <minute>0</minute>
          <second>0</second>
         </time>
        </property>
       </widget>
      </item>
      <item row="4" column="8">
       <widget class="QTimeEdit" name="endTimeSat">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="time">
         <time>
          <hour>22</hour>
          <minute>0</minute>
          <second>0</second>
         </time>
        </property>
       </widget>
      </item>
      <item row="4" column="6">
       <widget class="QTimeEdit" name="endTimeThu">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="time">
         <time>
          <hour>22</hour>
          <minute>0</minute>
          <second>0</second>
         </time>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QGridLayout" name="gridLayout_5">
      <item row="1" column="2">
       <widget class="QPushButton" name="close">
        <property name="text">
         <string>Close</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QPushButton" name="refresh">
        <property name="text">
         <string>Refresh</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QPushButton" name="apply">
        <property name="text">
         <string>Apply</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>buttonAccountState</tabstop>
  <tabstop>limitTimeAccess</tabstop>
  <tabstop>setTimePerDay</tabstop>
  <tabstop>monTime</tabstop>
  <tabstop>tueTime</tabstop>
  <tabstop>wedTime</tabstop>
  <tabstop>thuTime</tabstop>
  <tabstop>friTime</tabstop>
  <tabstop>satTime</tabstop>
  <tabstop>sunTime</tabstop>
  <tabstop>startTimeMon</tabstop>
  <tabstop>startTimeTue</tabstop>
  <tabstop>startTimeWed</tabstop>
  <tabstop>startTimeThu</tabstop>
  <tabstop>startTimeFri</tabstop>
  <tabstop>startTimeSat</tabstop>
  <tabstop>startTimeSun</tabstop>
  <tabstop>endTimeMon</tabstop>
  <tabstop>endTimeTue</tabstop>
  <tabstop>endTimeWed</tabstop>
  <tabstop>endTimeThu</tabstop>
  <tabstop>endTimeFri</tabstop>
  <tabstop>endTimeSat</tabstop>
  <tabstop>endTimeSun</tabstop>
  <tabstop>apply</tabstop>
  <tabstop>refresh</tabstop>
  <tabstop>close</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
